clang/tools/clang-format/git-clang-format

```diff
@@ -261,14 +267,14 @@ def main():
                 "ignored by clang-format):"
             )
             for filename in ignored_files:
-                print("    %s" % filename)
+                print_filename(filename, opts.null)
```

Comment:
```suggestion
            print_filenames(ignored_files, opts.print0)
```
and delete line 269 above.

Reply:
Done in ab12a7ef5c3942c0992d8ee3c1df42a29a114627.